sound/hybrid_font.h

```diff
@@ -120,7 +132,49 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (swing_player_) {
+        // avoid overlapping swings, based on value set in ProffieOSSwingOverlap.  Value is
+        // between 0 (no overlap) and 1.0 (full overlap)
+        if (swing_player_->pos() / swing_player_->length() >= config_.ProffieOSSwingOverlap) {
+          RefPtr<BufferedWavPlayer> overlap_swing = swing_player_;
+          swing_player_ = PlayPolyphonic(&swng);
+          overlap_swing->set_fade_time(overlap_swing->length() - overlap_swing->pos());
```

Comment:
Is this really needed? I mean, the swing sound has it's own natural fade-out, doesn't it?

Reply:
I’ll test without and see.

Reply:
It's required, odd-sounding cutoffs occur without it.